### PR TITLE
Add development time yarn db:clear:all script for easy analytics data clearing

### PR DESCRIPTION
**Pull request by rikukissa**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [x] I have linked the correct Github issue under "Development"
- [x] I have tested the changes locally, and written appropriate tests
- [x] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [x] I have updated the changelog with this change (if applicable)
- [x] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: infrastructure/clear-all-data.sh [hunk 1]
```diff
@@ -129,6 +129,7 @@ docker run --rm --network=$NETWORK --entrypoint=/bin/sh minio/mc:RELEASE.2025-05
 POSTGRES_DB="events"
 EVENTS_MIGRATOR_ROLE="events_migrator"
 EVENTS_APP_ROLE="events_app"
+ANALYTICS_POSTGRES_USER=${ANALYTICS_POSTGRES_USER:-"events_analytics"}
```
Comment (Zangetsu101) [inline infrastructure/clear-all-data.sh:132]: What's the role of this user in deployed environments or even locally? I just see it being dropped if it exists

File: infrastructure/clear-all-data.sh [hunk 2]
```diff
@@ -138,17 +140,28 @@ docker run --rm --network=$NETWORK  \
   -e POSTGRES_DB="${POSTGRES_DB}" \
   -e EVENTS_MIGRATOR_ROLE="${EVENTS_MIGRATOR_ROLE}" \
   -e EVENTS_APP_ROLE="${EVENTS_APP_ROLE}" \
+  -e ANALYTICS_POSTGRES_USER="${ANALYTICS_POSTGRES_USER}" \
```
Comment (Zangetsu101) [inline infrastructure/clear-all-data.sh]: The e2e repo needs any changes regarding this? I mean this will be different for each feature environment right?
